### PR TITLE
rename simulation library

### PR DESCRIPTION
**Pull request by xnetsrak**
## Description

rename IntegrationTesting project to SimulationLibrary
## References

* #0000

### CODE REVIEW

File: source/Energinet.DataHub.MessageHub.Client.SimpleInjector/documents/release-notes/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Energinet.DataHub.MessageHub.SimpleInjector Release notes
 
+## Version 3.2.1
+
+- Package and namespace renaming from IntegrationTesting to SimulationLibrary
```
Comment (defectiveAi) [inline source/Energinet.DataHub.MessageHub.Client.SimpleInjector/documents/release-notes/release-notes.md:5]: This is a breaking change, so version should be 3.3.0?

File: wordlist.txt [hunk 2]
```diff
@@ -125,4 +125,6 @@ bundleId
 MasterData
 masterdata
 TimeSeries
-bumbed
\ No newline at end of file
+bumbed
```
Comment (defectiveAi) [inline wordlist.txt:128]: 'bumbed'?

Comment (xnetsrak) [reply wordlist.txt:128]: I didn't add this one.

File: source/Energinet.DataHub.MessageHub.Client.SimpleInjector/documents/release-notes/release-notes.md (hunk 1 above)
Comment (xnetsrak) [reply source/Energinet.DataHub.MessageHub.Client.SimpleInjector/documents/release-notes/release-notes.md:5]: Yes maybe in the specific package with the name change. Not i the others. And it is a name.

Comment (defectiveAi) [reply source/Energinet.DataHub.MessageHub.Client.SimpleInjector/documents/release-notes/release-notes.md:5]: If teams upgrade to this version, their code would not compile. Dependabot used for tiny version updates will also break.

3.3.0 is da way.

Comment (xnetsrak) [reply source/Energinet.DataHub.MessageHub.Client.SimpleInjector/documents/release-notes/release-notes.md:5]: It is a new package. So nothing breaks.

**Review by defectiveAi — APPROVED** [review]

**Pull request merged**